src/AssetManager.php

```diff
@@ -244,6 +278,66 @@
         $this->jsFiles[$key]['attributes'] = $options;
     }
 
+    /**
+     * Registers a JavaScript code block.
+     *
+     * This method should be used for simple registration of JavaScript code blocks.
+     *
+     * @param string $jsString the JavaScript code block to be registered.
+     * @param array $options the HTML attributes for the script tag. The following options are specially handled and
+     * are not treated as HTML attributes:
+     *
+     * - `position`: specifies where the JS script tag should be inserted in a page. The possible values are:
+     *     * {@see \Yiisoft\View\WebView::POSITION_HEAD} in the head section
+     *     * {@see \Yiisoft\View\WebView::POSITION_BEGIN} at the beginning of the body section
+     *     * {@see \Yiisoft\View\WebView::POSITION_END} at the end of the body section. This is the default value.
+     *
+     * @param string|null $key the key that identifies the JS code block. If null, it will use $jsString as the key. If two JS code
+     * blocks are registered with the same key, the latter will overwrite the former.
+     *
+     * @return void
+     */
+    public function registerJsString(string $jsString, array $options = [], string $key = null): void
+    {
+        $key = $key ?: $jsString;
+
+        if (!\array_key_exists('position', $options)) {
+            $options = array_merge(['position' => 3], $options);
```

Comment:
What is 3?

Reply:
That is WebView::POSITION_END. As this is a literal copy from line 274, I will leave this for now for code consistency.

Reply:
After looking more into this, I guess this is done to not have to hard depend on the View package.

Reply:
I think it's not a problem to have the `view` as a dependency.
@samdark what do you think?

Reply:
See https://github.com/yiisoft/assets/commit/1fa9aa9efd2488ed22f4e43310f97333d9c90735